### ISSUE TITLE
Add Links to New Forum/Website

### ISSUE THREAD

**Issue by Deltium** [triplea-game/triplea#1502]
I'd like to emphasize the importance of connecting the online players with the new Warclub website (https://forums.triplea-game.org/).   In the next version of the software, there should be additional buttons to click on, like "Forum" and "Ladder" and "Tournaments" and these links would take players directly to this new website with the relevant content.  That would be cool and integrate both platforms nicely !

Please integrate this idea into the new software version.  It should not be too difficult, and we need to take advantage of the simple fact that the retention and growth of our online community could be enhanced by having a central place to exchange ideas, play PBEM, enter into tournaments and/or ladder games.

Thanks, Deltium

**Comment by RoiEXLab** [comment]
This is planned, once the forum is ready to be used...

**Comment by ron-murhammer** [comment]
Yep, agree with this.

**Comment by simon33-2** [comment]
Is this to replace the AxisandAllies.org forum?

**Comment by beelee1** [comment]
@simon33-2 
AxisandAllies.org is separate. It's a business

**Comment by DanVanAtta** [comment]
@RoiEXLab , can you elaborate on your plans? 
As far as forums, I thought we just needed to get an email server sorted out. (Let's follow up on gitter or in another issue on forum finalization)

We need to plan out how we want to integrate the two components. I think we're inviting trouble if we drive this from the website side first. The forums are good for authentication, but I think we should limit it there. The forum software is not suitable for driving tourneys.. I also would like to avoid a website feature that is completely divorced from the game engine.

I suggest we lead by first breaking up the requested features and implementing them in the game engine first. From there a read-only website should be pretty simple to get together. Then we can think about additional features and/or how we would want the website to interact back (eg: match acceptance)

So summarizing some of the really big questions  I see here:
- how do we think we'll want the lobby/game engine to integrate with the website from a technical perspective?
- should the website ever be more than a read-only view to the game engine data?   In what cases do we want to avoid having to launch the game before interacting with the central db?

**Comment by RoiEXLab** [comment]
@DanVanAtta I told you, that I expetimented with mail servers a bit. The problem I ran into, is that namecheap, the service the domain runs on, already has a mail server setup. (I get a no user found email when mailing to something@triplea-game.org) so you'd have to deal with that if we want that kind of domain!
I agree. Im planning on accessing the forum read only + Pbf, which is almost ready (file uploads not working).
We have a seperate db for the lobby. I'd like the lobby db to store any kind of stats regarding the lobby itself. We could add a system later, which enables us to "connect" a forum account to a lobby account showing the stats on the forum read only.
